CHANGELOG-0.10.md

```diff
@@ -0,0 +1,23 @@
+### Highlights
+
+- We've introduced two new custom resource definitions, `BackupStorageLocation` and `VolumeSnapshotLocation`, that replace the `Config` CRD from
+previous versions. As part of this, you may now configure more than one possible location for where backups and snapshots are stored, and when you
+create a `Backup` you can select the location where you'd like that particular backup to be stored.
+- Ark's plugin system has been significantly refactored to improve robustness and ease of development. Plugin processes are now automatically restarted
+if they unexpectedly terminate. Additionally, plugin binaries can now contain more than one plugin implementation (e.g. and object store *and* a block store,
+or many backup item actions).
+- The sync process, which ensures that Backup custom resources exist for each backup in object storage, has been revamped to run much more frequently (once
+per minute rather than once per hour), to use significantly fewer cloud provider API calls, and to not generate spurious Kubernetes API errors.
+- Ark can now be configured to store all data under a prefix within an object storage bucket. This means that you no longer need a separate bucket per Ark
+instance; you can now have all of your clusters' Ark backups go into a single bucket, with each cluster having its own prefix/subdirectory
+within that bucket.
+- Restic backup data is now automatically stored within the same bucket/prefix as the rest of the Ark data. A separate bucket is no longer required (or allowed).
+- Ark resources (backups, restores, schedules) can now be bulk-deleted through the `ark` CLI, using the `--all` or `--selector` flags, or by specifying 
+multiple resource names as arguments to the `delete` commands.
+- The `ark` CLI now supports waiting for backups and restores to complete with the `--wait` flag for `ark backup create` and `ark restore create`
+- Restores can be created directly from the most recent backup for a schedule, using `ark restore create --from-schedule SCHEDULE_NAME`
+
+### All New Features
+
+### Bug Fixes
+
```

Comment:
Given we know there are breaking changes, it might be worth adding a section calling that out specifically, and linking to the upgrade docs.

The highlights touch on it, but I think an explicit section informing people that just changing the image tag isn't enough is warranted.

Reply:
👍 was going to do that and forgot, thanks for the reminder.